modules/ROOT/pages/design-rename-project.adoc

```diff
@@ -0,0 +1,18 @@
+= Rename a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can rename an API-specification project or an API-fragment project.
+
+[NOTE]
```

Comment:
As with most Notes, this text could be converted to plain text and reduce "notice strain" to the audience.
See style ref. for more.

Reply:
Same as the Delete topic, this is a lot of formatting for what could be one simple text with a list.